packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/editingcontext/eventprocessorregistry/OnEditingContextEventProcessorRegistryTests.java

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.application.controllers.editingcontext.eventprocessorregistry;
+
+import java.util.Arrays;
+
+import org.eclipse.sirius.web.application.configurationproperties.SiriusWebProperties;
+import org.springframework.boot.autoconfigure.condition.ConditionMessage;
+import org.springframework.boot.autoconfigure.condition.ConditionOutcome;
+import org.springframework.boot.autoconfigure.condition.SpringBootCondition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+/**
+ * Custom condition used to detect that IEditingContextEventProcessorEditingContextProcessor test implementations should be enabled.
+ *
+ * @author arichard
+ */
+public class OnEditingContextEventProcessorRegistryTests extends SpringBootCondition {
```

Comment:
Those spring boot conditions are not exactly cheap on the long run in terms of build time, we have some to trigger some keep capabilities of Sirius Web during the tests but it's really overkill to create one to change the conditions of the tests (not the application) itself.